Tests/Common/Securities/RelativeStandardDeviationVolatilityModelTests.cs

```diff
@@ -178,5 +168,99 @@ public void GetHistoryRequirementsWorksForTwoDifferentSubscriptions()
             Assert.AreEqual(true, result.FillForwardResolution != null);
             Assert.AreEqual(true, result.IncludeExtendedMarketHours);
         }
+
+        [Test]
+        public void HandlesPriceDiscontinuityDueToSplit()
+        {
+            const int periods = 3;
+            var periodSpan = Time.OneMinute;
+            var model = new RelativeStandardDeviationVolatilityModel(periodSpan, periods);
+            var reference = new DateTime(2016, 04, 06, 12, 0, 0);
+
+            var security = GetSecurity(reference, model);
+
+            security.SetMarketPrice(new IndicatorDataPoint(reference, 1));
+            security.SetMarketPrice(new IndicatorDataPoint(reference.AddMinutes(1), 2));
+            security.SetMarketPrice(new IndicatorDataPoint(reference.AddMinutes(1.01), 1000));
+            security.SetMarketPrice(new IndicatorDataPoint(reference.AddMinutes(2), 3m));
+
+            var split = new Split(security.Symbol, reference.AddMinutes(2.5), 3m, 10m, SplitType.SplitOccurred);
+            model.ApplySplit(split, false, DataNormalizationMode.Raw);
+            security.SetMarketPrice(new IndicatorDataPoint(reference.AddMinutes(3), 30m));
+
+            var prices = new[] {20.0, 30.0, 30.0};
+            var expected = prices.StandardDeviation().SafeDecimalCast() / Math.Abs(prices.Mean().SafeDecimalCast());
+            Assert.AreEqual(expected, model.Volatility);
+        }
+
+        [Test]
+        public void HandlesPriceDiscontinuityDueToDividend()
+        {
+            const int periods = 3;
+            var periodSpan = Time.OneMinute;
+            var model = new RelativeStandardDeviationVolatilityModel(periodSpan, periods);
+            var reference = new DateTime(2016, 04, 06, 12, 0, 0);
+
+            var security = GetSecurity(reference, model);
+
+            security.SetMarketPrice(new IndicatorDataPoint(reference, 1));
+            security.SetMarketPrice(new IndicatorDataPoint(reference.AddMinutes(1), 2));
+            security.SetMarketPrice(new IndicatorDataPoint(reference.AddMinutes(1.01), 1000));
+            security.SetMarketPrice(new IndicatorDataPoint(reference.AddMinutes(2), 3m));
+
+            var dividend = new Dividend(security.Symbol, reference.AddMinutes(2.5), .031m, 3.1m);
+            model.ApplyDividend(dividend, false, DataNormalizationMode.Raw);
+            security.SetMarketPrice(new IndicatorDataPoint(reference.AddMinutes(3), 3m));
+
+            var prices = new[] { 2*.99, 3*.99, 3.0 };
+            var expected = prices.StandardDeviation().SafeDecimalCast() / Math.Abs(prices.Mean().SafeDecimalCast());
+            Assert.AreEqual(expected, model.Volatility);
+        }
+
+        [Test]
+        public void HandlesPriceDiscontinuityDueToSplitAndDividend()
+        {
+            const int periods = 3;
+            var periodSpan = Time.OneMinute;
+            var model = new RelativeStandardDeviationVolatilityModel(periodSpan, periods);
```

Comment:
Please add unit tests for every type of Indicator Volatility Model as well

Reply:
Tests for `IndicatorVolatilityModel` added.